__mocks__/fs.ts

```diff
@@ -1,2 +1,4 @@
 import { fs } from 'memfs';
 export const promises = fs.promises;
+
+export const readdirSync = jest.requireActual('fs').readdirSync;
```

Comment:
This was needed to get schema loading from the `@juptierone/data-model` working.